pkg/template/load_test.go

```diff
@@ -0,0 +1,102 @@
+package template
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestSplitYAML(t *testing.T) {
+	var tests = []struct {
+		desc     string
+		inBytes  []byte
+		outBytes [][]byte
+		err      error
+	}{
+		{
+			desc: "a simple YAML file with no leading document separator",
+			inBytes: []byte(`a: Easy
+b:
+    c: 2
+    d:
+        - 3
+        - 4
+`),
```

Comment:
```suggestion
			desc: "a simple YAML file with no leading document separator",
			inBytes: []byte(heredoc.Doc(`
				a: Easy
				b:
				  c: 2
				  d:
				    - 3
				    - 4
			`)),
```

Using [github.com/makenowjust/heredoc](https://github.com/makenowjust/heredoc) is a nice way to format blocks of text in test tables like this - it keeps the input reasonably indented with the rest of the test case and (in this particular example) means there isn't a dangling line at the start of the raw string, which is easy to miss.

Reply:
Ah nice thanks! Good to know. Will update accordingly :)